src/client.js

```diff
@@ -19,18 +20,50 @@ class SkynetClient {
    * @param {Object} [customOptions.params={}] - Query parameters to include in the URl.
    */
   constructor(portalUrl, customOptions = {}) {
+    // Check if portal URL provided twice.
+
     if (portalUrl && customOptions.portalUrl) {
       throw new Error(
         "Both 'portalUrl' parameter provided and 'customOptions.portalUrl' provided. Please pass only one in order to avoid conflicts."
       );
     }
 
+    // Add portal URL to options if given.
+
     this.customOptions = { ...customOptions };
     // If portal was not given, the default portal URL will be used.
     if (portalUrl) {
       // Set the portalUrl if given.
       this.customOptions.portalUrl = portalUrl;
     }
+
+    // Re-export selected client methods from skynet-js.
+
+    let browserClient = new BrowserSkynetClient(portalUrl);
+    this.browserClient = browserClient;
+
+    // Download
+    this.getSkylinkUrl = browserClient.getSkylinkUrl.bind(browserClient);
+
+    // File API
+    this.file = {
+      getEntryData: browserClient.file.getEntryData.bind(browserClient),
+      getEntryLink: browserClient.file.getEntryLink.bind(browserClient),
+    };
+
+    // SkyDB
+    this.db = {
+      setDataLink: browserClient.db.setDataLink.bind(browserClient),
+    };
+
+    // Registry
+    this.registry = {
+      getEntry: browserClient.registry.getEntry.bind(browserClient),
+      getEntryUrl: browserClient.registry.getEntryUrl.bind(browserClient),
+      getEntryLink: browserClient.registry.getEntryLink.bind(browserClient),
```

Comment:
Was this deprecated or we just added a non-async version?

Reply:
That is the deprecated async version. Maybe I could have left that out since we do export the sync version in index.js. Now that I think about it, this way will require the least amount of changes to existing scripts.